lerobot/common/policies/diffusion/modeling_diffusion.py

```diff
@@ -317,7 +317,9 @@ def __init__(self, config: DiffusionConfig):
         # Use a dry run to get the feature map shape.
         with torch.inference_mode():
             feat_map_shape = tuple(
-                self.backbone(torch.zeros(size=(1, *config.input_shapes["observation.image"]))).shape[1:]
+                self.backbone(
+                    torch.zeros(size=(1, config.input_shapes["observation.image"][0], *config.crop_shape))
+                ).shape[1:]
             )
```

Comment:
I broke it down. To answer your questions:

> And why do we use size=? It's quite unusual.

I just like being explicit with this arg because in numpy you sometimes don't have it first. My habit.

> Finally, I am wonder why we dont create our torch.zeros tensor with the device argument.

The device is not specified at this point so we default to CPU.